Utils/Hash/Hash.hpp

```diff
@@ -30,7 +30,7 @@ namespace Utils {
       // ----------------------------------------------------------------------
 
       //! The size type
-      using SizeType = NATIVE_INT_TYPE;
+      using SizeType = FwSizeType;
```

Comment:
It appears this `Hash::SizeType` is not actually used in anywhere in `Utils/Hash`. I would recommend either getting rid of it (my preference - not sure why it's there in the first place), or switch uses of `FwSizeType` to `Hash::SizeType`.

Reply:
Removed.